models/news.model.js

```diff
@@ -44,3 +44,18 @@ exports.userData = () => {
     return users;
   });
 };
+
+exports.articleData = () => {
```

Comment:
this function should have a bit more of a descriptive name as to what it's doing rather than just what it gets back